docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -1914,6 +1922,23 @@ Override the default architecture variant of the container image.
 
 This is equivalent to the Podman `--variant` option.
 
+## Quadlet section [Quadlet]
+Some quadlet specific configuration is shared between different unit types. Those settings
+can be configured in the `[Quadlet]` section.
+
+Valid options for `[Quadlet]` are listed below:
+
+| **[Quadlet] options**      | **Description**                                   |
+|----------------------------|---------------------------------------------------|
+| DefaultDependencies=false  | Include implicit network dependencies to the unit |
```

Comment:
The example sets the value to `false` but the description refers to the case of `true`. This might be a bit confusing

Reply:
Something got reverted here and the original issue is back. I'll take advantage and say that since setting it to `true` is redundant, it might be better to leave the `false` value and change the description to something like "Disable implicit network..."